src/js/edu-benefits/1990e/1990e-edu-benefits-entry.jsx

```diff
@@ -0,0 +1,35 @@
+import 'core-js';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { createHistory } from 'history';
+import { Router, useRouterHistory } from 'react-router';
+import { Provider } from 'react-redux';
+
+import initReact from '../../common/init-react';
+import routes from './routes';
+import reducer from './reducer';
+import createLoginWidget from '../../login/login-entry';
+import createCommonStore from '../../common/store';
+
+require('../../../sass/edu-benefits.scss');
+
+const store = createCommonStore(reducer);
+
+createLoginWidget(store);
+
+const browserHistory = useRouterHistory(createHistory)({
+  basename: '/education/apply-for-education-benefits/application'
```

Comment:
You probably need to add the form number to the base url here, and change it to '/' in the form config file.